test/fixtures/approvals/v3_tests.approved.json

```diff
@@ -0,0 +1,10 @@
+{
+  "problem_test": {
```

Comment:
I think the top-level key should match the resource. This under `/tracks/:id/exercise/...etc...` so let's make the top-level key `"exercise"`